interactions/api/models/guild.py

```diff
@@ -714,6 +753,283 @@ async def modify_member(
+        premium_progress_bar_enabled: Optional[bool] = None,
+        reason: Optional[str] = None,
+    ) -> "Guild":
+        """
+        Modifies the current guild.
+
+        :param name?: The new name of the guild
+        :type name: Optional[str]
+        :param verification_level?: The verification level of the guild
+        :type verification_level: Optional[VerificationLevel]
+        :param default_message_notifications?: The default message notification level for members
+        :type default_message_notifications: Optional[DefaultMessageNotificationLevel]
+        :param explicit_content_filter?: The explicit content filter level for media content
+        :type explicit_content_filter: Optional[ExplicitContentFilterLevel]
+        :param afk_channel_id?: The id for the afk voice channel
+        :type afk_channel_id: Optional[int]
+        :param afk_timeout?: Afk timeout in seconds
+        :type afk_timeout: Optional[int]
+        :param owner_id?: The id of the user to transfer the guild ownership to. You must be the owner to perform this
+        :type owner_id: Optional[int]
+        :param system_channel_id?: The id of the channel where guild notices such as welcome messages and boost events are posted
+        :type system_channel_id: Optional[int]
+        :param suppress_join_notifications?: Whether to suppress member join notifications in the system channel or not
+        :type suppress_join_notifications: Optional[bool]
+        :param suppress_premium_subscriptions?: Whether to suppress server boost notifications in the system channel or not
+        :type suppress_premium_subscriptions: Optional[bool]
+        :param suppress_guild_reminder_notifications?: Whether to suppress server setup tips in the system channel or not
+        :type suppress_guild_reminder_notifications: Optional[bool]
+        :param suppress_join_notification_replies?: Whether to hide member join sticker reply buttons in the system channel or not
+        :type suppress_join_notification_replies: Optional[bool]
+        :param rules_channel_id?: The id of the channel where guilds display rules and/or guidelines
+        :type rules_channel_id: Optional[int]
+        :param public_updates_channel_id?: The id of the channel where admins and moderators of community guilds receive notices from Discord
+        :type public_updates_channel_id: Optional[int]
+        :param preferred_locale?: The preferred locale of a community guild used in server discovery and notices from Discord; defaults to "en-US"
+        :type preferred_locale: Optional[str]
+        :param description?: The description for the guild, if the guild is discoverable
+        :type description: Optional[str]
+        :param premium_progress_bar_enabled?: Whether the guild's boost progress bar is enabled
+        :type premium_progress_bar_enabled: Optional[bool]
+        :param reason?: The reason for the modifying
+        :type reason: Optional[str]
+        """
```

Comment:
Please include `return` and `rtype`. Refer to the docstring variance from `send` in `CommandContext` to see how that appears.